src/Avalonia.Base/Animation/Animation.cs

```diff
@@ -358,14 +359,19 @@ public IDisposable Apply(Animatable control, IClock? clock, IObservable<bool> ma
             return new CompositeDisposable(subscriptions);
         }
 
+        public Task RunAsync(Animatable control) => RunAsync(control, null);
```

Comment:
Is it intentional to remove possibility to cancel animation from RunAsync? There were cancrllation token or IDisposable before.